temporalcli/commandsgen/commands.yml

```diff
@@ -195,6 +195,10 @@ commands:
       - name: no-json-shorthand-payloads
         type: bool
         description: Raw payload output, even if they are JSON.
+      - name: context-rpc-timeout
+        type: duration
+        description: Timeout for the underlying RPC call for a context call.
+        default: 20s
```

Comment:
Since Go SDK does [Deadline() / 2](https://github.com/temporalio/sdk-go/blob/master/internal/internal_utils.go#L129), setting default to 20s to match the Go SDK's default of 10s

Reply:
Actually, that can be misleading for the user. I'll instead not have a default for this param, and instead include a blurb in the description that the client defaults to 10s if unspecified